features/kernel/src/main/feature/feature.xml

```diff
@@ -114,13 +63,12 @@
     </feature>
 
     <!-- Override the pax-web pax-http-whiteboard feature and remove the vulnerable pax-web-jsp bundle-->
-    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="6.0.6_1">
+    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="${pax.web.version}_1">
         <feature version="[6.0,6.1)">pax-http</feature>
-        <bundle dependency="true" start-level="30">mvn:org.eclipse.jdt.core.compiler/ecj/4.4</bundle>
+        <bundle dependency="true" start-level="30">mvn:org.eclipse.jdt.core.compiler/ecj/4.5.1</bundle>
         <bundle start-level="30" dependency="true">mvn:javax.el/javax.el-api/3.0.0</bundle>
-        <!--<bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jsp/${project.version}</bundle>-->
-        <bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-extender-whiteboard/6.0.6</bundle>
-
+        <!--<bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jsp/${pax.web.version}</bundle>-->
+        <bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-extender-whiteboard/${pax.web.version}</bundle>
         <capability>
             osgi.contract;osgi.contract=JavaEl;version:Version="3";uses:="javax.el",
             osgi.contract;osig.contract=JavaJSP;version:Version="2.3";uses:="javax.servlet.jsp, javax.servlet.jsp.el, javax.servlet.jsp.tagext",
```

Comment:
Not related, but I let @clockard know as well that `osgi` is misspelled in `osig.contract`.